packs/bitbucket/rules/post_receive_webhook.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: "post_receive_webhook"
+pack: "bitbucket"
+description: "Trigger ST2 pack deploy for commit on Git branch"
+enabled: true
```

Comment:
It would be a good idea to disable this rule by default.

We disable rules which are part of common packs by default since they tend to be installation specific and user who wants to use it should explicitly enable it.